_high_availability/002-Replication-Configuration.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title:  "Replication configuration"
+date:   2015-01-27 22:02:36
+categories:
+permalink: /ha/replication.html
+---
+
+
+The master configuration
+=======================
+
+The master server is responsible for distributing the changes to the stand by servers. We have to change to following settings to achieve that:
+
+    port = 5234 # it's a good idea to change the port, we will use a connection pool 
```

Comment:
The use of pooling is orthogonal to this part of the guide. Settings of `port` is mostly a red herring.